streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -473,38 +496,34 @@ public void run() {
     }
 
     /**
-     * Produces a string representation containing useful information about Kafka Streams
-     * Such as thread IDs, task IDs and a representation of the topology. This is useful
-     * in debugging scenarios.
+     * Produce a string representation containing useful information about this {@code KafkaStream} instance such as
+     * thread IDs, task IDs, and a representation of the topology DAG including {@link StateStore}s (cf.
+     * {@link TopologyBuilder} and {@link KStreamBuilder}).
+     *
      * @return A string representation of the Kafka Streams instance.
      */
     @Override
     public String toString() {
-        return toString("");
-    }
-
-    /**
-     * Produces a string representation containing useful information about Kafka Streams
-     * such as thread IDs, task IDs and a representation of the topology starting with the given indent. This is useful
-     * in debugging scenarios.
-     * @return A string representation of the Kafka Streams instance.
-     */
-    public String toString(final String indent) {
-        final StringBuilder sb = new StringBuilder(indent + "KafkaStreams processID:" + processId + "\n");
+        final StringBuilder sb = new StringBuilder("\tKafkaStreams processID:" + processId + "\n");
         for (final StreamThread thread : threads) {
-            sb.append(thread.toString(indent + "\t"));
+            sb.append(thread.toString("\t\t"));
         }
         sb.append("\n");
 
         return sb.toString();
     }
 
     /**
-     * Cleans up local state store directory ({@code state.dir}), by deleting all data with regard to the application-id.
+     * Do a clean up of local {@link StateStore} directory ({@link StreamsConfig#STATE_DIR_CONFIG}), by deleting all
```

Comment:
I see.

I do not have a strong preference actually. But I remember we use singulars not plurals in many other classes and just wanted to be consistent. If it is actually the opposite case I'm happy to have them all to be plural.